### PR TITLE
change logging during upgrade

### PR DESCRIPTION
**Pull request by DaanHoogland**
### Description

This PR...

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [x] Cleanup (Code refactoring and cleanup, that may add test cases)

#### Feature/Enhancement Scale

- [ ] Major
- [x] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [ ] Major
- [ ] Minor
- [x] Trivial

### CODE REVIEW

**Review by nvazquez — APPROVED** [review]
LGTM

**Comment by nvazquez** [comment]
@blueorangutan package

File: engine/schema/src/main/java/com/cloud/upgrade/dao/Upgrade410to420.java [hunk 1]
```diff
@@ -2388,19 +2388,32 @@ private void migrateDatafromIsoIdInVolumesTable(Connection conn) {
                                 conn.prepareStatement("ALTER TABLE `cloud`.`volumes` CHANGE COLUMN `iso_id1` `iso_id` bigint(20) unsigned COMMENT 'The id of the iso from which the volume was created'");) {
                             alter_iso_pstmt.executeUpdate();
                         }catch (SQLException e) {
-                            s_logger.error("migrateDatafromIsoIdInVolumesTable:Exception:"+e.getMessage(),e);
+                            s_logger.info("migrateDatafromIsoIdInVolumesTable: ignoring Exception:"+e.getMessage());
```
Comment (sureshanaparti) [inline engine/schema/src/main/java/com/cloud/upgrade/dao/Upgrade410to420.java:2391]:
```suggestion
                            s_logger.info("migrateDatafromIsoIdInVolumesTable: ignoring Exception:" + e.getMessage());
```

File: engine/schema/src/main/java/com/cloud/upgrade/dao/Upgrade410to420.java [hunk 2]
```diff
@@ -2388,19 +2388,32 @@ private void migrateDatafromIsoIdInVolumesTable(Connection conn) {
...
+                            if (s_logger.isTraceEnabled()) {
+                                s_logger.trace("migrateDatafromIsoIdInVolumesTable: ignored Exception",e);
+                            }
                             //implies iso_id1 is not present, so do nothing.
                         }
                     }catch (SQLException e) {
-                        s_logger.error("migrateDatafromIsoIdInVolumesTable:Exception:"+e.getMessage(),e);
+                        s_logger.info("migrateDatafromIsoIdInVolumesTable: ignoring Exception:"+e.getMessage());
```
Comment (sureshanaparti) [inline engine/schema/src/main/java/com/cloud/upgrade/dao/Upgrade410to420.java:2398]:
```suggestion
                        s_logger.info("migrateDatafromIsoIdInVolumesTable: ignoring Exception: " + e.getMessage());
```

File: engine/schema/src/main/java/com/cloud/upgrade/dao/Upgrade410to420.java [hunk 3]
```diff
@@ -2388,19 +2388,32 @@ private void migrateDatafromIsoIdInVolumesTable(Connection conn) {
...
+                        if (s_logger.isTraceEnabled()) {
+                            s_logger.trace("migrateDatafromIsoIdInVolumesTable: ignored Exception",e);
+                        }
                         //implies iso_id1 is not present, so do nothing.
                     }
                 }
             }catch (SQLException e) {
-                s_logger.error("migrateDatafromIsoIdInVolumesTable:Exception:"+e.getMessage(),e);
+                s_logger.info("migrateDatafromIsoIdInVolumesTable: ignoring Exception:"+e.getMessage());
```
Comment (sureshanaparti) [inline engine/schema/src/main/java/com/cloud/upgrade/dao/Upgrade410to420.java:2406]:
```suggestion
                s_logger.info("migrateDatafromIsoIdInVolumesTable: ignoring Exception: " + e.getMessage());
```

File: engine/schema/src/main/java/com/cloud/upgrade/dao/Upgrade410to420.java [hunk 4]
```diff
@@ -2388,19 +2388,32 @@ private void migrateDatafromIsoIdInVolumesTable(Connection conn) {
...
+                if (s_logger.isTraceEnabled()) {
+                    s_logger.trace("migrateDatafromIsoIdInVolumesTable: ignored Exception",e);
+                }
                 //implies iso_id1 is not present, so do nothing.
             }
         } catch (SQLException e) {
+          s_logger.info("migrateDatafromIsoIdInVolumesTable: ignoring Exception:"+e.getMessage());
```
Comment (sureshanaparti) [inline engine/schema/src/main/java/com/cloud/upgrade/dao/Upgrade410to420.java:2413]:
```suggestion
          s_logger.info("migrateDatafromIsoIdInVolumesTable: ignoring Exception: " + e.getMessage());
```

**Review by sureshanaparti — APPROVED** [review]
clgtm, left some suggestions for code formatting. please check.

**Comment by nvazquez** [comment]
@blueorangutan package

**Comment by nvazquez** [comment]
No need to run tests on this PR, simple logging refactor

**Pull request merged**